Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -93,6 +94,14 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UFUNCTION(BlueprintCallable, Category="C++")
 	float GetDifficultyMultiplier();
 
+	/** Removes progression system spot that should not be available by other objects anymore. */
+	UFUNCTION(BlueprintCallable, Category = "C++")
+	void RemoveProgressionSystemSpot(UPSSpotComponent* ProgressionSystemSpotComponent);
+	
+	/** Destroy all star actors that should not be available by other objects anymore. */
+	UFUNCTION(BlueprintCallable, Category = "C++")
+	void DestroyStarActors();
```

Comment:
- I would do `DestroyStarActors` under `protected` within `meta = (BlueprintProtected)`, as only World subsystem should be responsible for handling own leaks
- Please add `All` to the name to make it more clear like `DestroyAllStarActors` 
- Update comment, that it's called mainly only once on unloading to cleanup all star actors, all the rest of time stars are pooling, but never destroying

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/3a6669199b8ad56e31faf19f76dde2b89fa5f61a